AFQ/segmentation.py

```diff
@@ -762,20 +779,20 @@ def move_streamlines(self, tg=None, reg_algo='slr'):
             self.logger.info("Registering tractogram with SLR")
             atlas = self.bundle_dict['whole_brain']
             self.moved_sl, _, _, _ = whole_brain_slr(
-                atlas, self.tg.streamlines, x0='affine', verbose=False,
+                atlas, tg.streamlines, x0='affine', verbose=False,
```

Comment:
Maybe we should change the signature of this function to the following?

```
def move_streamlines(self, tg, reg_algo='slr')
```
It looks like you made it so that the `tg` argument is never going to be optional.